Dockerfile

```diff
@@ -64,4 +64,10 @@ EXPOSE 8080
 #Setup docker entrypoint and start the nginx service
 COPY ./entrypoint.sh /
 RUN chmod +x /entrypoint.sh
-ENTRYPOINT ["/entrypoint.sh"]
\ No newline at end of file
+ENTRYPOINT ["/entrypoint.sh"]
+
+
+# Set the locale
+ENV LANG=C.UTF-8
+ENV LANGUAGE=C.UTF-8
+
```

Comment:
Let's also add
ENV LC_ALL=C.UTF-8
So, that it also set all the other locale variables.
Example  - https://newbedev.com/how-to-set-the-locale-inside-a-debian-ubuntu-docker-container

Reply:
Added